tests/test_user_flow.py

```diff
@@ -87,3 +102,17 @@ def it_requires_defendant_name(self, post_suit):
         errors = post_suit({}).soup.find_all(class_='error')
         assert errors[0].parent.parent.find('label').text == \
             'Your Brother\'s Full Name'
+
+
+class WhenNavigatingToAdminUsers(object):
+
+    @patch('lib.oidc_old.OIDC.openid_config', mock_openid_config)
+    def it_redirects_to_identity_broker(self,
+                                        client,
+                                        test_admin_user,
+                                        admin_logged_in):
```

Comment:
Avoid arbitrary indentation. Break the line at the opening bracket and indent two levels for function/method parameters